jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -248,7 +249,7 @@ private void generateSchema(Class<?> schemaClass) throws MojoExecutionException
         JsonNode jsonSchema = getGenerator().generateSchema(schemaClass);
         File file = getSchemaFile(schemaClass);
         this.getLog().info("- Writing schema to file: " + file);
-        this.writeToFile(jsonSchema, file);
+        this.writeToFile(getGenerator().getObjectMapper(), jsonSchema, file);
```

Comment:
**suggestion**: As per the other comment, let's only expose a `getConfig()` getter on the generator.
```suggestion
        this.writeToFile(getGenerator().getConfig().getObjectMapper(), jsonSchema, file);
```